CIP-Liesenfelt-Shelleys_Voltaire_decentralization_update/README.md

```diff
@@ -48,19 +48,27 @@ The Cardano network currently produces ~21,600 blocks per epoch with ~2400 group
 
 (2) https://iohk.io/en/blog/posts/2018/10/29/preventing-sybil-attacks/
 
-<img src="equation1.png" width="400"> (1)
+$ Ref (1) $
+$$ \frac{1}{k_{effective}} = \sum_{group \space g}^{G(~2436)} (weight_g)\bigg(\frac{1}{k_g}\bigg) $$
+
+$$ \frac{1}{k_{effective}} = \sum_{group \space g}^{G(~2436)} \bigg(\frac{group \space stake_g}{total \space stake}\bigg) \bigg(\frac{group \space stake_g}{total \space stake}\bigg)  $$
+
+$$ \frac{1}{k_{effective}} = \sum_{group \space g}^{G(~2436)} \bigg(\frac{group \space stake_g}{total \space stake}\bigg)^2 $$
```

Comment:
I might have to revert these markdown equations back to your images where it doesn't render.  Apparently github does not support LaTex really yet, but soon.  It renders in VS Code or Jupyter Notebooks though.  The easiest fix is to use your images.

Reply:
fixed equation rendering